test/language/identifiers/start-unicode-15.0.0-escaped.js

```diff
@@ -0,0 +1,16 @@
2}\u{32263}\u{32264}\u{32265}\u{32266}\u{32267}\u{32268}\u{32269}\u{3226A}\u{3226B}\u{3226C}\u{3226D}\u{3226E}\u{3226F}\u{32270}\u{32271}\u{32272}\u{32273}\u{32274}\u{32275}\u{32276}\u{32277}\u{32278}\u{32279}\u{3227A}\u{3227B}\u{3227C}\u{3227D}\u{3227E}\u{3227F}\u{32280}\u{32281}\u{32282}\u{32283}\u{32284}\u{32285}\u{32286}\u{32287}\u{32288}\u{32289}\u{3228A}\u{3228B}\u{3228C}\u{3228D}\u{3228E}\u{3228F}\u{32290}\u{32291}\u{32292}\u{32293}\u{32294}\u{32295}\u{32296}\u{32297}\u{32298}\u{32299}\u{3229A}\u{3229B}\u{3229C}\u{3229D}\u{3229E}\u{3229F}\u{322A0}\u{322A1}\u{322A2}\u{322A3}\u{322A4}\u{322A5}\u{322A6}\u{322A7}\u{322A8}\u{322A9}\u{322AA}\u{322AB}\u{322AC}\u{322AD}\u{322AE}\u{322AF}\u{322B0}\u{322B1}\u{322B2}\u{322B3}\u{322B4}\u{322B5}\u{322B6}\u{322B7}\u{322B8}\u{322B9}\u{322BA}\u{322BB}\u{322BC}\u{322BD}\u{322BE}\u{322BF}\u{322C0}\u{322C1}\u{322C2}\u{322C3}\u{322C4}\u{322C5}\u{322C6}\u{322C7}\u{322C8}\u{322C9}\u{322CA}\u{322CB}\u{322CC}\u{322CD}\u{322CE}\u{322CF}\u{322D0}\u{322D1}\u{322D2}\u{322D3}\u{322D4}\u{322D5}\u{322D6}\u{322D7}\u{322D8}\u{322D9}\u{322DA}\u{322DB}\u{322DC}\u{322DD}\u{322DE}\u{322DF}\u{322E0}\u{322E1}\u{322E2}\u{322E3}\u{322E4}\u{322E5}\u{322E6}\u{322E7}\u{322E8}\u{322E9}\u{322EA}\u{322EB}\u{322EC}\u{322ED}\u{322EE}\u{322EF}\u{322F0}\u{322F1}\u{322F2}\u{322F3}\u{322F4}\u{322F5}\u{322F6}\u{322F7}\u{322F8}\u{322F9}\u{322FA}\u{322FB}\u{322FC}\u{322FD}\u{322FE}\u{322FF}\u{32300}\u{32301}\u{32302}\u{32303}\u{32304}\u{32305}\u{32306}\u{32307}\u{32308}\u{32309}\u{3230A}\u{3230B}\u{3230C}\u{3230D}\u{3230E}\u{3230F}\u{32310}\u{32311}\u{32312}\u{32313}\u{32314}\u{32315}\u{32316}\u{32317}\u{32318}\u{32319}\u{3231A}\u{3231B}\u{3231C}\u{3231D}\u{3231E}\u{3231F}\u{32320}\u{32321}\u{32322}\u{32323}\u{32324}\u{32325}\u{32326}\u{32327}\u{32328}\u{32329}\u{3232A}\u{3232B}\u{3232C}\u{3232D}\u{3232E}\u{3232F}\u{32330}\u{32331}\u{32332}\u{32333}\u{32334}\u{32335}\u{32336}\u{32337}\u{32338}\u{32339}\u{3233A}\u{3233B}\u{3233C}\u{3233D}\u{3233E}\u{3233F}\u{32340}\u{32341}\u{32342}\u{32343}\u{32344}\u{32345}\u{32346}\u{32347}\u{32348}\u{32349}\u{3234A}\u{3234B}\u{3234C}\u{3234D}\u{3234E}\u{3234F}\u{32350}\u{32351}\u{32352}\u{32353}\u{32354}\u{32355}\u{32356}\u{32357}\u{32358}\u{32359}\u{3235A}\u{3235B}\u{3235C}\u{3235D}\u{3235E}\u{3235F}\u{32360}\u{32361}\u{32362}\u{32363}\u{32364}\u{32365}\u{32366}\u{32367}\u{32368}\u{32369}\u{3236A}\u{3236B}\u{3236C}\u{3236D}\u{3236E}\u{3236F}\u{32370}\u{32371}\u{32372}\u{32373}\u{32374}\u{32375}\u{32376}\u{32377}\u{32378}\u{32379}\u{3237A}\u{3237B}\u{3237C}\u{3237D}\u{3237E}\u{3237F}\u{32380}\u{32381}\u{32382}\u{32383}\u{32384}\u{32385}\u{32386}\u{32387}\u{32388}\u{32389}\u{3238A}\u{3238B}\u{3238C}\u{3238D}\u{3238E}\u{3238F}\u{32390}\u{32391}\u{32392}\u{32393}\u{32394}\u{32395}\u{32396}\u{32397}\u{32398}\u{32399}\u{3239A}\u{3239B}\u{3239C}\u{3239D}\u{3239E}\u{3239F}\u{323A0}\u{323A1}\u{323A2}\u{323A3}\u{323A4}\u{323A5}\u{323A6}\u{323A7}\u{323A8}\u{323A9}\u{323AA}\u{323AB}\u{323AC}\u{323AD}\u{323AE}\u{323AF};
```

Comment:
Isn't this test testing only that `\u{1123F}` is accepted as an identifier start character? This test would pass even if `\u{11240}` is only accepted as an identifier continuation character.

Reply:
Good point! Fixed in https://github.com/mathiasbynens/caniunicode/commit/49b9c03a63256b909bad5d1b01ce727b6473a0db and pushed to this PR.